src/lib.rs

```diff
@@ -26,13 +26,17 @@ pub struct WalOptions {
     /// The number of segments to create ahead of time, so that appends never
     /// need to wait on creating a new segment.
     pub segment_queue_len: usize,
+
+    /// Keep these many "closed-*" wal files around. Defaults to 1.
+    pub max_closed_to_preserve: usize,
```

Comment:
I vote to rename this to something like `retain_closed`. I find 'max' and 'preserve' a bit confusing in this context.

Reply:
Great naming :)